integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/schema/CarbonAlterTableDropColumnCommand.scala

```diff
@@ -108,6 +99,15 @@ private[sql] case class CarbonAlterTableDropColumnCommand(
         }
       }
 
+      // Check if column to be dropped is of complex dataType
+      alterTableDropColumnModel.columns.foreach { column =>
+        if (carbonTable.getColumnByName(alterTableDropColumnModel.tableName, column).getDataType
```

Comment:
Can we merge this loop and the loop on line 83

Reply:
done.. please review